.dockerignore

```diff
@@ -1,2 +1,6 @@
 .env
-Dockerfile
\ No newline at end of file
+Dockerfile
+.terraform.lock.hcl
```

Comment:
I wouldn't necessarily disagree with this in `.dockerignore` but actually it's best to leave it out of `.gitignore` - it specifies which versions of providers we're using, so like our `poetry.lock` it helps keep environments consistent.

[Hashicorp's docs state](https://developer.hashicorp.com/terraform/language/files/dependency-lock#lock-file-location):
> You should include this file in your version control repository so that you can discuss potential changes to your external dependencies via code review, just as you would discuss potential changes to your configuration itself.